bin/run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# This script provides the funcionality to use the service for 
+# testing purposes inside a Docker container. It allows you to
+#
+# - Initialize the testing database when INTEGRATION_TEST_SETUP is set
+# - Start the service
+#
+set -Eeuxo pipefail
```

Comment:
```suggestion
# Default behaviour in bash is that a line with a failure doesnt break the script, but it carries on. With this set of flags, if there is an error during the script it will break; this minimise unexpected behaviours in the script during a failure (like a fail in the condition check that carries on and runs the next line).
set -Eeuxo pipefail
```